bootstrap/sql/mysql/create_tables.sql

```diff
@@ -38,11 +38,14 @@ CREATE TABLE IF NOT EXISTS schema_version_info (
   version          INT                   NOT NULL,
   schemaMetadataId BIGINT                NOT NULL,
   timestamp        BIGINT                NOT NULL,
+  state            TINYINT               NOT NULL,
```

Comment:
Really need upgrade scripts, the amount of changes in the db now, is getting dangerous for any users with existing data, having to custom upgrade their DB without any upgrade scripts that also are tested.